src/Telemetry/AzureDevOpsTimeline/AzureDevOpsTimeline.cs

```diff
@@ -45,6 +45,10 @@ public async Task<TimeSpan> RunAsync(CancellationToken cancellationToken)
         {
             TraceSourceManager.SetTraceVerbosityForAll(TraceVerbosity.Fatal);
 
+            Console.WriteLine($"Config BuildBatchSize: {_options.Value.BuildBatchSize}");
```

Comment:
We DEFINITELY shouldn't be Console.WriteLine the access token.  (The other stuff probably isn't that interesting either, but if you wanted to _log.LogInfo it, that would be fine)